docs/spec/tuples.rst

```diff
@@ -0,0 +1,154 @@
+Tuples
+======
+
+The ``tuple`` class has some special behaviors and properties that make it
+different from other classes from a typing perspective. The most obvious
+difference is that ``tuple`` is variadic -- it supports an arbitrary number
+of type arguments. Each type argument represents a single element, and those
+elements are immutable at runtime. These properties affect subtyping rules and
```

Comment:
```suggestion
of type arguments. In fixed-sized tuple types, each type argument represents
a single element. These properties affect subtyping rules and
```

It's not strictly accurate that the elements are immutable, since a tuple can contain mutable objects. The statement seems unnecessary anyway, as the immutability of tuples is a property of the language and doesn't have to be repeated here.

It's also not accurate that each type arguments represents a single element (in `tuple[int, ...]`, `int` represents an arbitrary number of elements). The suggested wording fixes that issue but is still concise.